static/js/admin/src/MoveDropLocation.js

```diff
@@ -140,21 +139,81 @@ define('MoveDropLocation', [ 'jquery', 'jquery-ui', 'ConfirmationDialog'],
 	};
 	
 	MoveDropLocation.prototype.setMoveActive = function(active) {
-		if (active) {
-			this.element.addClass("moving");
-			this.element.on("click.dropClickBlocking", "a", function(e) {
-				e.preventDefault();
-			}).on("mouseenter.dropTargetHover", this.options.dropTargetSelector, function() {
-				$(this).addClass("drop_hover");
-			}).on("mouseleave.dropTargetLeave", this.options.dropTargetSelector, function() {
-				$(this).removeClass("drop_hover");
-			});
-		} else {
-			this.element.removeClass("moving");
-			this.element.off("click.dropClickBlocking").off("mouseenter.dropTargetHover").off("mouseleave.dropTargetLeave");
+		var targets = $('.structure_content a.res_link, .result_table a.res_link');
+
+		try {
+			if (active) {
+				var self = this;
+				var adminUnits = [];
+
+				// Highlight valid drop locations
+				targets.each(function() {
```

Comment:
A few suggestions for this loop:
1. Loop through all the dragTargets before this loop to determine if there are any admin units being moved. That would allow things to fail quickly, and its not something that will vary by potential destination. You could then move the adminUnitsMoving check out of the `targets` loop as well.
2. Initialize `destInfo` once per target, probably right after `var selector = $(this);` since it will not vary by dragTarget.
3. You can move the setting of `targets` down to after adminUnit check. Also, could you rename it to something like "destinations" so that the name isn't so similar to `dragTargets`?
4. If the current `target` is an invalid destination for a `dragTarget`, then there's no need to check anymore `dragTargets` against that destination.